cpp/src/arrow/compute/kernels/hash_aggregate.cc

```diff
@@ -1737,6 +1737,12 @@ struct AntiExtrema {
   static constexpr CType anti_max() { return std::numeric_limits<CType>::min(); }
 };
 
+template <>
+struct AntiExtrema<bool> {
+  static constexpr float anti_min() { return true; }
+  static constexpr float anti_max() { return false; }
```

Comment:
I'm curious, why is float returned?

Reply:
Because I don't pay attention when I copy-paste, probably…fixed.